test/Microsoft.AspNetCore.SystemWebAdapters.Tests/VirtualPathUtilityTests.cs

```diff
@@ -0,0 +1,212 @@
+using System;
+using System.Web;
+using Xunit;
+
+namespace Microsoft.AspNetCore.SystemWebAdapters.Tests
+{
+    public class VirtualPathUtilityTests
+    {
+        [InlineData("/", "/")]
+        [InlineData("~/", "/")]
+        [Theory]
+        public void ToAbsolute(string virtualPath, string expected)
+        {
+            Assert.Equal(expected, (string?)VirtualPathUtility.ToAbsolute(virtualPath, "/"));
+        }
+
+        [Fact]
+        public void ToAbsoluteError()
+        {
+#if NET40_OR_GREATER
+			// This does not match the documentation
+			Assert.Throws<ArgumentException>(() => VirtualPathUtility.ToAbsolute("hello", "/"));
```

Comment:
Is there a way to make the exceptions match?

Reply:
I can make the exceptions match - just net4x doesn't throw the exception that the docs site says it does and I went by the docs in this case.
https://docs.microsoft.com/en-us/dotnet/api/system.web.virtualpathutility.toabsolute?view=netframework-4.8 says it throws an `ArgumentOutOfRangeException` but the framework actually throws an `ArgumentException` (or sometimes an `ArgumentNullException`)

Reply:
I've updated the code to match the same exceptions now. I think I also caught one more scenario which wasn't being caught while I was at it and removed the resource file.